themes/default/content/cloud-engineering-summit/_index.md

```diff
@@ -0,0 +1,37 @@
+---
+# TODO: update these values
+title: Cloud Engineering Summit
+meta_desc: The Cloud Engineering Summit is a day of learning for cloud practitioners about cloud infrastructure, modern applications, and everything in between.
+
+type: page
+layout: cloud-engineering-summit
+
+tracks_outline:
+    - name: Build
+      topics:
+        - Reusable infrastructure practices
+        - Docker, containers, or Kubernetes
+        - Building React Native, cloud-based apps
+        - Serverless apps development
+        - Data science and ML apps
+        - Sharing your open source project
+      description: |
+        Modern cloud applications and architectures blur the lines between app infrastructure and cloud engineering teams adopting new tools and practices to tame complexity and accelerate delivery. They embrace that success in the modern cloud era requires some level of proficiency in both cloud infrastructure and applications. Whether they're full-stack or specialize in one area, all cloud engineers apply a software engineering mindset and practices to building and testing applications and the underlying cloud infrastructure. This includes using standard programming languages, applying software principles such as reusability and abstractions and testing, and leveraging the rich ecosystem of software development tools.
```

Comment:
Wording here seems like something may be missing -- is this maybe right?

```suggestion
        Modern cloud applications and architectures blur the lines between app infrastructure and cloud engineering teams adopting new tools and practices to tame complexity and accelerate delivery. Embracing that success in the modern cloud era requires some level of proficiency in both cloud infrastructure and applications. Whether they're full-stack or specialize in one area, all cloud engineers apply a software engineering mindset and practices to building and testing applications and the underlying cloud infrastructure. This includes using standard programming languages, applying software principles such as reusability and abstractions and testing, and leveraging the rich ecosystem of software development tools.
```

Reply:
👍